src/hooks/useCreateTargetForm.js

```diff
@@ -2,11 +2,13 @@ import { useState, useEffect } from 'react';
 import { useStatus } from '@rootstrap/redux-tools';
 
 import { createTarget, getTopics } from 'actions/targetActions';
+import strings from 'locale';
 
 const useCreateTargetForm = (topic, emptyTopic, handleSubmit) => {
-  const { error: targetError, status: targetStatus } = useStatus(createTarget);
+  const { error, status: targetStatus } = useStatus(createTarget);
   const { error: getTopicsError, status: getTopicsStatus } = useStatus(getTopics);
   const [topicError, setTopicError] = useState();
+  const targetError = error ? strings.TARGET.maximumTargetsError : '';
```

Comment:
If there is another kind of error this will display always the maximum target message, right?

Reply:
I think the same